Assets/Scripts/UnitController.cs

```diff
@@ -16,6 +16,22 @@ public class UnitController : EntityController
     public int unitArmyIndex = 0;
     public Vector2Int unitSquadIndex = new Vector2Int(-1, -1);
 
+    [Header("Sprite characteristics")]
+
+    public bool isBorderActive = false;
+
+    [SerializeField]
```

Comment:
Surprisingly this solution of having 4 sprites in the background gives a suprisingly good effect, but I'd suggest you try either one of the following two options:
- Outline shader, you can use ShaderGraph to make it, maybe [this video](https://www.youtube.com/watch?v=84rZ-rCRsZk) can help
- Or a simpler method, render a "selection" sprite at the feet of the unit. Example of this effect in Minecraft Legends 
![Minecraft Legends](https://github.com/hsven/mystic-militia/assets/72400318/3ba44e55-9ac3-44e8-aaf8-6e93600b49c4)

I ask this since it will be easier to use with new sprites in the future.

Reply:
I understand, and I started by trying with shader and it was not working. As we need this feature as soon as possible to develop I used duplicate sprites.

I think we can put it as an issue and wait a bit before do it

Reply:
Honestly I think this (or at least, the second method) is a rather straightforward thing to implement, but sure it can be dealt with later